ClosedXML/Excel/PivotTables/PivotStyleFormats/XLPivotStyleFormat.cs

```diff
@@ -0,0 +1,40 @@
+// Keep this file CodeMaid organised and cleaned
+using System.Collections.Generic;
+
+namespace ClosedXML.Excel
+{
+    internal class XLPivotStyleFormat : IXLPivotStyleFormat
+    {
+        public XLPivotStyleFormat()
+            : this(null, XLStyle.Default)
+        {
+        }
+
+        public XLPivotStyleFormat(IXLPivotField field)
+            : this(field, XLStyle.Default)
+        { }
+
+        public XLPivotStyleFormat(IXLStyle style)
+            : this(null, style)
+        { }
+
+        public XLPivotStyleFormat(IXLPivotField field, IXLStyle style)
```

Comment:
Out of 4 public constructors, one is never used and the other is used just once. Given that the class is internal, what do you think about decreasing the number of constructors? See also `XLPivotValueStyleFormat`.

Reply:
Hos about this?

```
public XLPivotStyleFormat(IXLPivotField field = null, IXLStyle style = XLStyleDefault)
{
  PivotField = field;
  Style = style ?? XLStyle.Default;
}
```

Reply:
Habit of mine that when I create the class I create all possible constructors I think would be used.